Makefile

```diff
@@ -87,7 +87,7 @@ validate: lint
 	@BUILDTAGS="$(BUILDTAGS)" hack/validate.sh
 
 lint:
-	$(GOBIN)/golangci-lint run --build-tags "$(BUILDTAGS)"
```

Comment:
I also worry that moving the build tags to the config is going to get out of sync.